src/job_cache/daemon_cache.cpp

```diff
@@ -1140,8 +1139,20 @@ void DaemonCache::launch_evict_loop() {
 
     // Finally enter the eviction loop, if it exits cleanly
     // go ahead and exit with its result.
-    int result =
-        eviction_loop(".", std::make_unique<LRUEvictionPolicy>(max_cache_size, low_cache_size));
+    std::unique_ptr<EvictionPolicy> policy;
+    switch (config.type) {
+      case EvictionPolicyType::TTL:
+        wcl::log::info("Using TTL eviction policy, seconds_to_live = %lu",
+                       config.ttl.seconds_to_live)();
+        policy = std::make_unique<TTLEvictionPolicy>(config.ttl.seconds_to_live);
+        break;
+      case EvictionPolicyType::LRU:
+        wcl::log::info("Using LRU eviction policy, max = %lu, min = %lu", config.lru.low_size,
```

Comment:
It looks like the debug print has the max and min switched.

Reply:
It does indeed, I'll fix it in a follow up before we release